src/NServiceBus.AcceptanceTests/Routing/When_doing_content_based_routing_with_commands.cs

```diff
@@ -0,0 +1,137 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using Configuration.AdvanceExtensibility;
+    using EndpointTemplates;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+    using Settings;
+
+    public class When_doing_content_based_routing_with_commands : NServiceBusAcceptanceTest
```

Comment:
I think the naming is to specific to the implementation instead of relating to the used core APIs. From a core perspective it could make more sense to name it something like `When_using_custom_routing_strategy` ?